### PR TITLE
WIP: Disable H/W offload for encapsulated network devices

### PR DESCRIPTION
**Pull request by randomvariable**
Signed-off-by: Naadir Jeewa <jeewan@vmware.com>

### CODE REVIEW

File: images/capi/ansible/roles/node/files/etc/udev/rules.d/90-netif-disable-hw-offload.rules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+ACTION=="add", SUBSYSTEM=="net", NAME=="veth*", TAG+="netif_hw_offload_disable"
+ACTION=="add", SUBSYSTEM=="net", NAME=="tunl", TAG+="netif_hw_offload_disable"
```
Comment (dims) [inline images/capi/ansible/roles/node/files/etc/udev/rules.d/90-netif-disable-hw-offload.rules:2]: is this just `"tunl"` or should it be `"tunl*"` as well?

Comment (randomvariable) [reply images/capi/ansible/roles/node/files/etc/udev/rules.d/90-netif-disable-hw-offload.rules:2]: I'll need to check, but I think it's just `tunl`.

**Comment by vincepri** [comment]
Can we add a condition that would allow us to easily disable this patch using some configuration option?

**Comment by randomvariable** [comment]
Closing this.

Basically, I am seeing other L3 routing issues even with TX/RX offload disabled when IPIP encap is on.

**Pull request closed without merging**